stan/math/prim/prob/wiener_lpdf.hpp

```diff
@@ -138,15 +138,18 @@ return_type_t<T_y, T_alpha, T_tau, T_beta, T_delta> wiener_lpdf(
     return 0;
   }
 
-  static const double WIENER_ERR = 0.000001;
-  static const double PI_TIMES_WIENER_ERR = pi() * WIENER_ERR;
-  static const double LOG_PI_LOG_WIENER_ERR = LOG_PI + log(WIENER_ERR);
-  static const double TWO_TIMES_SQRT_TWO_PI_TIMES_WIENER_ERR
+  static constexpr double WIENER_ERR = 0.000001;
+  static constexpr double PI_TIMES_WIENER_ERR = pi() * WIENER_ERR;
+  static constexpr double LOG_PI_LOG_WIENER_ERR
+      = LOG_PI + -13.81551055796427410410794872810618524;
```

Comment:
```suggestion
  static constexpr double LOG_PI_LOG_WIENER_ERR
      = LOG_PI + -6 * boost::math::constants::ln_ten<double>();
```

Might be worth adding `LOG_TEN` as a constant to `constants.hpp`